api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java

```diff
@@ -23,7 +23,6 @@
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public class KafkaBridgeAdminClientSpec extends KafkaBridgeClientSpec {
-    private static final long serialVersionUID = 1L;
 
```

Comment:
As I said before - please stop creating the empty lines at the top of the file. This applies to most of the files here.

Reply:
Guess it is something what could be added to checkstyle?

Reply:
Yeah, sorry I forgot about that :/ updated 👍

Reply:
Also there is another problem that our checkstyle ignores more lines between import groups f.e., 
```

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import io.strimzi.api.kafka.model.common.Constants;
import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
import io.strimzi.crdgenerator.annotations.Description;
import io.strimzi.crdgenerator.annotations.DescriptionFile;
import io.strimzi.crdgenerator.annotations.Minimum;
import io.sundr.builder.annotations.Buildable;
import lombok.EqualsAndHashCode;
import lombok.ToString;


import java.util.HashMap;
import java.util.Map;
```
this treats as okay. Which is not good FPOV. I would update check style :
```diff
- <property name="separateLineBetweenGroups" value="false"/>
+ <property name="separateLineBetweenGroups" value="true"/>
```